src/lib/components/colors/Header.svelte

```diff
@@ -171,6 +179,24 @@
     grid-area: swatch;
     height: 100%;
     position: relative;
+    &.out-of-gamut {
+      outline: var(--warning) 3pt solid;
+      outline-offset: 1pt;
+    }
+
+    .gamut-warning {
+      background: var(--warning);
+      border-radius: var(--border-radius);
+      color: var(--bg);
+      display: flex;
+      font-size: var(--small);
+      gap: var(--half-shim);
+      margin: var(--shim) 0 0 var(--shim);
+      outline: var(--border-width) solid var(--bg);
+      padding: var(--half-shim) var(--shim);
+      position: absolute;
+      z-index: 1;
```

Comment:
do we want to use the z-index utility here instead? It will have the same result, but would be nice if we have add any further index numbers.

Reply:
good catch. fixed in https://github.com/oddbird/oddcontrast/pull/226/commits/b065585643624174ccfe01ecd19cc9ad5b17ddaf